app/helpers/template_helpers.py

```diff
@@ -41,116 +160,69 @@ def get_page_header_context(language: str, theme: str) -> Dict[str, Any]:
-            [
-                {
-                    "text": lazy_gettext("Languages"),
-                    "url": static_content_urls["languages"],
-                    "target": "_blank",
-                },
-                {
-                    "text": lazy_gettext("BSL and audio videos"),
-                    "url": static_content_urls["bsl_and_audio_videos"],
-                    "target": "_blank",
-                },
-            ]
-        )
-
-    default_context = {
-        "lang": language_code,
-        "crest": True,
-        "newTabWarning": lazy_gettext("The following links open in a new tab"),
-        "copyrightDeclaration": {
-            "copyright": lazy_gettext(
-                "Crown copyright and database rights 2021 NIMA MOU577.501."
-            )
-            if theme == "census-nisra"
-            else lazy_gettext("Crown copyright and database rights 2020 OS 100019153."),
-            "text": lazy_gettext(
-                "Use of address data is subject to the terms and conditions."
-            ),
-        },
-        "rows": [
-            {"itemsList": items_list},
-        ],
-        "legal": [
-            {
-                "itemsList": [
-                    {
-                        "text": lazy_gettext("Cookies"),
-                        "url": static_content_urls["cookies"],
-                        "target": "_blank",
-                    },
-                    {
-                        "text": lazy_gettext("Accessibility statement"),
-                        "url": static_content_urls["accessibility_statement"],
-                        "target": "_blank",
-                    },
-                    {
-                        "text": lazy_gettext("Privacy and data protection"),
-                        "url": static_content_urls["privacy_and_data_protection"],
-                        "target": "_blank",
-                    },
-                    {
-                        "text": lazy_gettext("Terms and conditions"),
-                        "url": static_content_urls["terms_and_conditions"],
-                        "target": "_blank",
-                    },
-                ],
-            },
-        ],
-    }
+        }
 
-    if request.blueprint == "post_submission":
-        default_context["footerWarning"] = lazy_gettext(
-            "Make sure you <a href='{sign_out_url}'>leave this page</a> or close your browser if using a shared device"
-        ).format(sign_out_url=sign_out_url)
+    def footer_context(self, sign_out_url: Optional[str] = None) -> Dict[str, Any]:
+        default_context = super().footer_context(sign_out_url=sign_out_url)
 
-    context = {
-        "default": {
-            **default_context,
-        },
-        "social": {
-            **default_context,
-        },
-        "northernireland": {
-            **default_context,
-        },
-        "census": {
-            **default_context,
-        },
-        "census-nisra": {
+        context = {
```

Comment:
```suggestion
        return {
            **default_context,
            "copyrightDeclaration": {
                "copyright": lazy_gettext(
                    "Crown copyright and database rights 2021 NIMA MOU577.501."
                ),
                "text": lazy_gettext(
                    "Use of address data is subject to the terms and conditions."
                ),
            },
            "lang": "en",
            "poweredBy": {
                "logo": "nisra-logo-black-en",
                "alt": "NISRA - Northern Ireland Statistics and Research Agency",
            },
        }
```

Reply:
This has changed slightly, so an early return is no longer an option